xla/stream_executor/rocm/rocm_dnn.h

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 #include "absl/synchronization/mutex.h"
 #include "absl/types/span.h"
 #include "rocm/include/miopen/miopen.h"
+#include "xla/stream_executor/device_memory_allocator.h"
```

Comment:
I think this needs a corresponding BUILD file update

Reply:
It builds as-is.

Reply:
Not in a strict mode. With layering check enabled, for every header include you need to have a corresponding dependency to a target that has it listed in its hdrs section.

Reply:
So the target you would need to depend on here is //xla/stream_executor:device_memory_allocator